src/SDKs/_metadata/storage_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-06-25 03:10:42 UTC
+2018-08-01 03:28:26 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: azure
 Branch:      master
-Commit:      37ca9e71d6c89b46ab990eb064ddb70308fd5824
+Commit:      1e2f0dae5c44d55b642d27232f9dcd7c0876488a
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\weiwei\AppData\Roaming\npm `-- autorest@2.0.4280  
```

Comment:
Please pull down the latest changes from psSdkJson6, run `msbuild build.proj` and regenerate the code